etl_framework/config_mixins/FiltersMixin.py

```diff
@@ -29,6 +30,26 @@ def add_filters(self, filter_mappings):
             self.set_filter(getattr(filter_mappings, self.get_filter()))
             self.set_post_filter(getattr(filter_mappings, self.get_post_filter()))
 
+    def add_filters_from_module(self, filter_module):
+        """
+        :summary: takes in a filter module and matches filters from it to
+        filters mentioned in config
+        :param filter_module: mpython module where filters reside
+        """
+        super(FiltersMixin, self).add_filters(filter_module)
+
+        pre_filter = getattr(filter_module, self.get_pre_filter())
```

Comment:
@ianlofs you'll still get the same error we were getting before if `self.get_pre_filter()` returns `None`. `getattr(filter_module, None)` throws a TypeError because it expects a string instead of None.

In this case, we just want to set pre_filter to None (it never gets called because the transformer doesn't call it)